GitUI/UserControls/FileStatusList.cs

```diff
@@ -1282,6 +1326,25 @@ private void FileStatusListView_ContextMenu_Opening(object sender, CancelEventAr
                 cm.Items.Add(new ToolStripSeparator());
                 cm.Items.Add(_sortByContextMenu);
             }
+
+            if (!cm.Items.Find(ShowAllDiferencesItemName, true).Any())
```

Comment:
This menu appears also in other forms, can be confusing there
The tooltip is good here, but I am leaning on removing the option anyway

Reply:
A little more information please